### ISSUE TITLE
Generated code triggers clippy lint for Rust 1.63

### ISSUE THREAD

**Issue by 9names** [rust-embedded/svd2rust#655]
Rust 1.63 clippy added a lint that is being triggered by svd2rust output.
The lint is
`clippy::derive_partial_eq_without_eq`

This is what the clippy warning looks like:

```
warning: you are deriving `PartialEq` and can implement `Eq`
   --> src/usbctrl_regs/sie_status.rs:632:30
    |
632 | #[derive(Clone, Copy, Debug, PartialEq)]
    |                              ^^^^^^^^^ help: consider deriving `Eq` as well: `PartialEq, Eq`
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#derive_partial_eq_without_eq
```

Are we able to add this derive to the generated output?
